lms/static/scripts/frontend_apps/components/BookPicker.tsx

```diff
@@ -69,43 +69,53 @@ export default function BookPicker({
   const canSubmit =
     (step === 'select-book' && book) || (step === 'select-chapter' && chapter);
 
-  // Provide a spacious modal UI when listing chapters (taller modal)
-  const showChapters = step === 'select-chapter' && !error;
+  /**
+   * Prevent submission of content form, which can raise an error because
+   * content has not been selected.
+   */
+  const onCancelButtonClick = (event: Event) => {
+    event.preventDefault();
+    event.stopPropagation();
+    onCancel();
+  };
 
   return (
     <Modal
       // Opt out of Modal's automatic focus handling; route focus manually in
       // sub-components
-      initialFocus={null}
-      onCancel={onCancel}
-      contentClass={classnames('LMS-Dialog LMS-Dialog--wide', {
-        'LMS-Dialog--tall': showChapters,
-      })}
+      initialFocus={'manual'}
```

Comment:
NB: You can review props or other documentation about the updated `Modal` in LMS' own Pattern library if you're running LMS locally: http://localhost:8001/ui-playground/feedback-modal